configs/server/forseti_conf_server.yaml.in

```diff
@@ -207,6 +207,8 @@ scanner:
           enabled: false
         - name: ke_version_scanner
           enabled: true
+        - name: kms_scanner
```

Comment:
This is more like crypto key scanner instead of KMS? KMS has keyring and cryptokey and seems like we are only addressing crypto key in this scanner.

Reply:
I believe this scanner was named KMS scanner to be more generic. 
@blueandgold do we want to change the name from KMS scanner to crypto key scanner?